HW_1/Task_3.py

```diff
@@ -0,0 +1,23 @@
+# Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+lower_limit = 0
+upper_limit = 100000
+
+while True:
+    num = int(input("Введите положительное число не более 100 тысяч: "))
+    if lower_limit < num > lower_limit:
+        break
+    print("Введено неверное число. Попробуйте еще раз")
+
+if num == 1:
+    print("Число 1 не простое и не составное")
+if num == 2:
+    print("Число 2 - простое")
+
+for i in range(2, num):
+    if num % i == 0:
+        print("Число ", num, " - составное")
+        break
+    if i == num - 1:
```

Comment:
Используйте связку `break` и `else` от цикла с выводом соответствующего сообщения. Условие при этом не нужно
```suggestion
else:
```